migration/20180313-add-identifier-id-to-materialized-view.sql

```diff
@@ -0,0 +1,9 @@
+DO $$
+    BEGIN
+        BEGIN
+	    create index mv_works_for_lanes_identifier_id on mv_works_for_lanes (identifier_id);
+        EXCEPTION
+            WHEN duplicate_table THEN RAISE NOTICE 'Warning: mv_works_for_lanes_identifier_id already exists.';
+        END;
+    END
+$$;
```

Comment:
In my tests this index was not totally necessary, in that the query was acceptably fast without it, but I kept it in because the performance improvement was significant.